aws/README.md

```diff
@@ -20,6 +19,19 @@ Have the following tools installed:
 
 Make sure you have an active account at AWS for which you have configured the credentials on the system where you will execute the steps below. In this example we stored the credentials under an aws profile as `awsuser`.
 
+### Multi-user setup: shared state
+
+If you want to host a multi-user setup, you will probably want to share the state file so that everyone can try related challenges. We have provided a starter to easily do so using a Terraform S3 backend.
+
+First, create an s3 bucket:
+
+```bash
+cd shared-state
+terraform apply
+```
+
+The bucket name should be in the output. Please use that to configure the terraform backend in `main.tf`.
+
```

Comment:
Do we want to include a note that "a multi-user setup" is possible from a centraly reachable server as well?

Reply:
In what sense? If you initialise and apply the terraform from a centrally reachable server in another way?

Reply:
yes: because then people can see the terraform state on that centralized server ....I thought..

Reply:
Definitely true, but I don't really think that's something we want to make a 'paved road' in terms of support. So I'd vouch for keeping this as is until we see that's a deployment strategy people use/really want

Reply:
maybe good to mention in our upcoming talks?Let's resolve this one for now.